packages/lxd/src/file/push.coffee.md

```diff
@@ -41,7 +44,7 @@ console.info(`File was pushed: ${$status}`)
             type: 'boolean'
             default: false
             description: '''
-            Create any directories necessary.
+            Container side. Create any directories necessary in the container.
```

Comment:
Remove "Container side", it is not a sentence, you already write that it creates directories in the container